src/versions/v1/SandySounds.js

```diff
@@ -306,7 +311,7 @@ class SandySounds extends EventEmitter {
     }
 
 
-    async voiceServerUpdate(data) {
+    async voiceServerUpdate(data, session_id) {
```

Comment:
no need to put it as a seperate param when you can just attach it to the data object

Reply:
@MrJohnCoder It's more convenient this way.

It's easier to do this:
```javascript
return this.voice.voiceServerUpdate(packet.d, sessionId);
```

Than this:
```javascript
req.packet.d.session_id = this.sessionid
return this.voice.voiceServerUpdate(req.packet.d);
```